week2/homework/src/index1.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+// TODO: Write the homework code in this file
+const fs = require('fs');
+const readFile = fs.readFileSync('./data.json', 'utf8');
+const readindex = fs.readFileSync('./index.txt', 'utf8');
+var json = JSON.parse(readFile);
+
+
+const cmd = process.argv[2]
+const todo = process.argv[3];
+const todo2 = process.argv[4];
+switch (cmd) {
+    case 'add':
+        add(todo);
+        break;
+    case 'remove':
+        remove(todo)
+        break;
+    case 'list':
+        todolist()
+        break;
+    case 'reset':
+        reset()
+        break;
+    case 'update':
```

Comment:
If I remove the third item with `node . remove 3` and then do `node . update 3 "asdasd"` a new item is created (with index 3), so the index is not updated if I remove an item. 
Maybe an easy solution could be to remove the id (number) altogether and just use an array, in that way the index updating is done automatically. In your json file you would have something like
```
todos: [
    { 
        "text": "shoping", 
        "done": false
    },
    {
        "text": "lunch",
        "done": false
    }
]
```
If you then want the first, you can do todos[0] and if you delete that one the second becomes the first automatically.